exercises/secret-handshake/src/main/java/HandshakeCalculator.java

```diff
@@ -1,5 +1,9 @@
-final class HandshakeCalculator {
+import java.util.List;
 
+final class HandshakeCalculator {
 
+    List<Signal> calculateHandshake(int number) {
```

Comment:
I suggest adding the `public` modifier. It's not necessary for the tests to pass, but it's not generally left out, unless you specifically want something to be package private.